igor-web/src/main/groovy/com/netflix/spinnaker/igor/scm/ScmInfoController.groovy

```diff
@@ -46,6 +50,9 @@ class ScmInfoController {
 
         if(gitHubMaster)
             result << [gitHub : gitHubMaster.baseUrl]
+
+        if(bitBucketMaster)
```

Comment:
I am not sure this change is needed. It looks like we have a list of github masters but bitbucket cloud api only allows one endpoint.